src/main/java/cz/codecamp/logger/loggers/WithoutTimeLogger.java

```diff
@@ -0,0 +1,14 @@
+package cz.codecamp.logger.loggers;
+
+import cz.codecamp.logger.FormatterInterface;
+import cz.codecamp.logger.LogLevelEnum;
+
+public class WithoutTimeLogger implements FormatterInterface {
```

Comment:
Jelikož je to implementace `FormatterInterface`, mělo by být v package `formatters` a mít postfix `*Formatter`.